### ISSUE TITLE
Coverage Report

### ISSUE THREAD

**Issue by rickhedin** [practicalmeteor/meteor-munit#30]
Hi.  Is there some way of getting a report "You've tested 30% of the lines in your program"?  Similar to Blanket.  Thanks, Rick

**Comment by rbabayoff** [comment]
No, not that I know off. The people at velocity are working on something like that, but not for packages, for now. It will require package build plugins.
